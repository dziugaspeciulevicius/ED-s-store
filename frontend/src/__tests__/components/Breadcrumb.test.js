import React from "react";
import { shallow } from "enzyme";
import "../../setupTests";
import Breadcrumb from "../../components/Breadcrumb.component";

describe("Breadcrumb tests", () => {
  it("Renders <Breadcrumb /> component correctly", () => {
    // shallow creates instance of a component
    expect(shallow(<Breadcrumb />)).toMatchSnapshot();
  });
});
