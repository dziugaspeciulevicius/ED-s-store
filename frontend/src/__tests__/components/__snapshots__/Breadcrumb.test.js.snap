// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlaceOrderPage /> Renders <PlaceOrderPage /> component correctly 1`] = `ShallowWrapper {}`;

exports[`Breadcrumb tests Renders <Breadcrumb /> component correctly 1`] = `ShallowWrapper {}`;
